nuxeo-features/nuxeo-automation/nuxeo-automation-core/src/main/java/org/nuxeo/ecm/automation/core/operations/document/RetrieveFromColdStorage.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * (C) Copyright 2020 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *  
+ * Contributors:
+ *     Salem Aouana
+ */
+
+package org.nuxeo.ecm.automation.core.operations.document;
+
+import org.nuxeo.ecm.automation.core.Constants;
+import org.nuxeo.ecm.automation.core.annotations.Context;
+import org.nuxeo.ecm.automation.core.annotations.Operation;
+import org.nuxeo.ecm.automation.core.annotations.OperationMethod;
+import org.nuxeo.ecm.automation.core.annotations.Param;
+import org.nuxeo.ecm.automation.core.collectors.DocumentModelCollector;
+import org.nuxeo.ecm.core.api.Blob;
+import org.nuxeo.ecm.core.api.CoreSession;
+import org.nuxeo.ecm.core.api.DocumentModel;
+import org.nuxeo.ecm.core.blob.ColdStorageHelper;
+
+/**
+ * Retrieve the cold storage content associated with the input {@link DocumentModel}. This operation will initiate a
+ * request restore, calling the {@link Blob#getStream()} during this process doesn't mean you will get the blob content
+ * immediately.
+ * 
+ * @since 11.1
+ */
+@Operation(id = RetrieveFromColdStorage.ID, category = Constants.CAT_BLOB, label = "Request a retrieve from Cold Storage", description = "Request a cold storage content retrieval associated with the document. ")
```

Comment:
We should try to be consistent: if we consider that the "retrieve" operation/helper performs a restore (restoration?) request, let's always use these terms.

Reply:
yep, i asked on slack initially when i started the devs on this ticket and i introduced the term `restore` to be closer with the S3 documentation where mainly they used the term `restore`.

- what if the rename the retrieve operation/helper -> restore operation/helper
- or if we want to keep the word `retrieve` i will update the label/description of the operation

Reply:
I would keep retrieve and update the operation label to "Request restoration from cold storage" and description "Request a cold storage content restoration associated with the document".